### DISCUSSION TITLE
U1 Dexcom G6 slot

### DISCUSSION THREAD

**Discussion by TheCassos** [NightscoutFoundation/xDrip#3507] [Q&A]
Hey Guys
Just wondering if its possible to select Dexcom G6 transmitter slot (phone/medical) like the GTS
we don't use a pump, so we usually put the Phone in the Phone slot, Watch in Medical slot
option appears to be greyed out in u1 xdrip dexcom settings
Many thanks!
Jerry

**Comment by Navid200** [comment]
Are you using G6 or G7?

Just enter the transmitter ID in U1.
Use the phone as you always do with engineering mode disabled.
U1 is supposed to handle it itself.
There is no need to enter the BlueJay menu neither on the phone nor on U1.

You say you have no pump.  So, even with GTS, you are not supposed to manipulate the slots.
